### ISSUE TITLE
Any experiments or checkpoint on the VAE model

### ISSUE THREAD

**Issue by zihan0822** [Laurafdez/VAE-MODEL#1]
Have you guys done any experiments on this repo. Does it work well?  Just want to make sure its usability before I fork and work on it.

**Comment by Laurafdez** [comment]
Yes!! It should work fine. If you encounter any issues, please don't hesitate to ask for assistance. I'm here to help. ;)
